docs/community/styleguide/formatting.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Formatting
+description: This guide illustrates the standards for formatting and writing our documentation.
+weight: 2
+---
+## Documentation formatting
+
```

Comment:
Please add a paragraph or sentence introducing the topic "Documentation formatting".